lib/cadet_web/router.ex

```diff
@@ -125,6 +116,11 @@ defmodule CadetWeb.Router do
   scope "/v2", CadetWeb do
     pipe_through([:api, :auth, :ensure_auth])
 
+    get("/assessments", AssessmentsController, :index)
+    post("/assessments/:assessmentid", AssessmentsController, :show)
+    post("/assessments/:assessmentid/submit", AssessmentsController, :submit)
+    post("/assessments/question/:questionid/submit", AnswerController, :submit)
```

Comment:
Hmm.. this feels like it should be `/assessments/questions/:questionid/answer`&mdash;reason being you are posting the answer for that question. (Sorry if I didn't raise this earlier.) Also note the plural `questions`.